packages/odyssey-react-mui/scripts/properties-to-json.js

```diff
@@ -0,0 +1,100 @@
+/*!
+ * Copyright (c) 2023-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+const { resolve, join, basename, extname } = require("node:path");
+const properties = require("properties");
+const {
+  ensureDirSync,
+  ensureFileSync,
+  readFileSync,
+  writeFileSync,
+  removeSync,
+} = require("fs-extra");
+const readdir = require("recursive-readdir");
+
+const convert = (baseFiles, propertiesTargetDir) => {
+  for (const src of baseFiles) {
+    const filename = basename(src);
+    const extension = extname(src);
+    const source = `${readFileSync(src)}`;
+
+    properties.parse(source, function (error, obj) {
+      // eslint-disable-next-line no-console
+      if (error) {
+        return console.error(error);
+      }
+
+      const targetFile = join(
+        propertiesTargetDir,
+        filename.replace(extension, ".ts")
+      );
+      ensureFileSync(targetFile);
+      writeFileSync(
+        targetFile,
+        "/* eslint-disable header/header */\nexport const translation = " +
+          JSON.stringify(obj) +
+          ";"
```

Comment:
Disabling the header/header rule for the generated files. Also had to add the path to these to `.prettierignore`

Reply:
Just add the header. It's copy-paste, but that's the easiest way to ensure it's there. We even do it in our generated icons.

Unless these are generated at compile-time and not comitted, then that's different.

Reply:
They will be, trying to add these to git ignore